frontend/routes/package/versions.tsx

```diff
@@ -254,6 +260,19 @@ function Version({
             </button>
           </form>
         )}
+        {isPublished && iam.isStaff && (
```

Comment:
Does this only show up when sudo is on?

Reply:
good catch, fixed in https://github.com/jsr-io/jsr/pull/1027